charts/operator-wandb/charts/flat-run-fields-updater/templates/deployment.yaml

```diff
@@ -97,18 +97,6 @@ spec:
               value: "mysql://$(MYSQL_USER):$(MYSQL_PASSWORD)@$(MYSQL_HOST):$(MYSQL_PORT)/$(MYSQL_DATABASE)"
             - name: GORILLA_RUN_STORE
```

Comment:
let's also include `GORILLA_RUN_STORE` with the `wandb_local_flat` in the release channels, so that this is enabled only for managed installs.